components/expanders/expander.js

```diff
@@ -0,0 +1,72 @@
+const expander = {
+  init: function() {
+    this.cacheDom()
+    this.bindEvents()
+  },
+  cacheDom: function() {
+    this.expanderComponents = document.querySelectorAll('.msds-expander__list-item')
+    this.expanderSections = document.querySelectorAll('.msds-expander--section-expander')
+  },
+  bindEvents: function() {
+    this.expanderComponents.forEach(function() {
+      document.body.onkeyup = expander.setActiveExpanderOnEnterKey
+    })
+
+    for (const expanderSection of this.expanderSections) {
+      expanderSection.onclick = expander.setActiveSection
+    }
+  },
+  setActiveExpanderOnEnterKey: function(event) {
+    const enterKeyCode = 13
+
+    if (event.keyCode == enterKeyCode) {
+      const expanderChildren = event.target.childNodes
+
+      expanderChildren.forEach(function(expandingElement) {
+        const isExpanded = expandingElement.checked
+        const elementType = expandingElement.nodeName
+        if (elementType == 'INPUT' && isExpanded == true) {
+          expander.openExpander(expandingElement)
+        } else if (elementType == 'INPUT' && isExpanded == false) {
+          expander.closeExpander(expandingElement)
+        }
```

Comment:
The same long comment like for accordions - this logic should be in the parent component in my opinion.